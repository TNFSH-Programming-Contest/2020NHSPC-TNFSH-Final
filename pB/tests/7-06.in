506
904.4 -419.8
909.5 -408.6
914.5 -397.2
919.4 -385.9
924.1 -374.4
928.7 -362.9
933.1 -351.3
937.4 -339.7
941.6 -328.1
945.6 -316.4
949.4 -304.6
953.1 -292.8
956.7 -280.9
960.1 -269.0
963.4 -257.1
966.5 -245.1
969.5 -233.1
972.3 -221.0
975.0 -208.9
977.5 -196.8
979.9 -184.6
982.1 -172.5
984.1 -160.3
986.0 -148.0
987.8 -135.8
989.4 -123.5
990.9 -111.2
992.2 -98.9
993.3 -86.6
994.3 -74.2
995.2 -61.9
995.9 -49.5
996.4 -37.1
996.8 -24.8
997.0 -12.4
997.1 0.0
997.0 12.4
996.8 24.8
996.4 37.1
995.9 49.5
995.2 61.9
994.3 74.2
993.3 86.6
992.2 98.9
990.9 111.2
989.4 123.5
987.8 135.8
986.0 148.0
984.1 160.3
982.1 172.5
979.9 184.6
977.5 196.8
975.0 208.9
972.3 221.0
969.5 233.1
966.5 245.1
963.4 257.1
960.1 269.0
956.7 280.9
953.1 292.8
949.4 304.6
945.6 316.4
941.6 328.1
937.4 339.7
933.1 351.3
928.7 362.9
924.1 374.4
919.4 385.9
914.5 397.2
909.5 408.6
904.4 419.8
899.1 431.0
893.7 442.2
888.1 453.2
882.4 464.2
876.6 475.1
870.6 486.0
864.5 496.8
858.3 507.5
851.9 518.1
845.4 528.6
838.8 539.1
832.1 549.4
825.2 559.7
818.2 569.9
811.0 580.1
803.7 590.1
796.4 600.0
788.8 609.9
781.2 619.6
773.5 629.3
765.6 638.8
757.6 648.3
749.5 657.6
741.3 666.9
732.9 676.0
724.5 685.1
715.9 694.0
707.2 702.9
698.5 711.6
689.6 720.2
680.6 728.7
671.5 737.1
662.3 745.4
653.0 753.6
643.6 761.6
634.0 769.5
624.4 777.4
614.7 785.0
604.9 792.6
595.1 800.1
585.1 807.4
575.0 814.6
564.8 821.7
554.6 828.6
544.3 835.4
533.9 842.1
523.4 848.7
512.8 855.1
502.1 861.4
491.4 867.6
480.6 873.6
469.7 879.5
458.7 885.3
447.7 890.9
436.6 896.4
425.4 901.8
414.2 907.0
402.9 912.1
391.6 917.0
380.1 921.8
368.7 926.4
357.1 930.9
345.5 935.3
333.9 939.5
322.2 943.6
310.5 947.5
298.7 951.3
286.8 954.9
275.0 958.4
263.0 961.8
251.1 965.0
239.1 968.0
227.0 970.9
215.0 973.6
202.9 976.2
190.7 978.7
178.6 981.0
166.4 983.1
154.1 985.1
141.9 986.9
129.6 988.6
117.3 990.2
105.0 991.5
92.7 992.8
80.4 993.9
68.0 994.8
55.7 995.5
43.3 996.2
30.9 996.6
18.6 996.9
6.2 997.1
-6.2 997.1
-18.6 996.9
-30.9 996.6
-43.3 996.2
-55.7 995.5
-68.0 994.8
-80.4 993.9
-92.7 992.8
-105.0 991.5
-117.3 990.2
-129.6 988.6
-141.9 986.9
-154.1 985.1
-166.4 983.1
-178.6 981.0
-190.7 978.7
-202.9 976.2
-215.0 973.6
-227.0 970.9
-239.1 968.0
-251.1 965.0
-263.0 961.8
-275.0 958.4
-286.8 954.9
-298.7 951.3
-310.5 947.5
-322.2 943.6
-333.9 939.5
-345.5 935.3
-357.1 930.9
-368.7 926.4
-380.1 921.8
-391.6 917.0
-402.9 912.1
-414.2 907.0
-425.4 901.8
-436.6 896.4
-447.7 890.9
-458.7 885.3
-469.7 879.5
-480.6 873.6
-491.4 867.6
-502.1 861.4
-512.8 855.1
-523.4 848.7
-533.9 842.1
-544.3 835.4
-554.6 828.6
-564.8 821.7
-575.0 814.6
-585.1 807.4
-595.1 800.1
-604.9 792.6
-614.7 785.0
-624.4 777.4
-634.0 769.5
-643.6 761.6
-653.0 753.6
-662.3 745.4
-671.5 737.1
-680.6 728.7
-689.6 720.2
-698.5 711.6
-707.2 702.9
-715.9 694.0
-724.5 685.1
-732.9 676.0
-741.3 666.9
-749.5 657.6
-757.6 648.3
-765.6 638.8
-773.5 629.3
-781.2 619.6
-788.8 609.9
-796.4 600.0
-803.7 590.1
-811.0 580.1
-818.2 569.9
-825.2 559.7
-832.1 549.4
-838.8 539.1
-845.4 528.6
-851.9 518.1
-858.3 507.5
-864.5 496.8
-870.6 486.0
-876.6 475.1
-882.4 464.2
-888.1 453.2
-893.7 442.2
-899.1 431.0
-904.4 419.8
-909.5 408.6
-914.5 397.2
-919.4 385.9
-924.1 374.4
-928.7 362.9
-933.1 351.3
-937.4 339.7
-941.6 328.1
-945.6 316.4
-949.4 304.6
-953.1 292.8
-956.7 280.9
-960.1 269.0
-963.4 257.1
-966.5 245.1
-969.5 233.1
-972.3 221.0
-975.0 208.9
-977.5 196.8
-979.9 184.6
-982.1 172.5
-984.1 160.3
-986.0 148.0
-987.8 135.8
-989.4 123.5
-990.9 111.2
-992.2 98.9
-993.3 86.6
-994.3 74.2
-995.2 61.9
-995.9 49.5
-996.4 37.1
-996.8 24.8
-997.0 12.4
-997.1 0.0
-997.0 -12.4
-996.8 -24.8
-996.4 -37.1
-995.9 -49.5
-995.2 -61.9
-994.3 -74.2
-993.3 -86.6
-992.2 -98.9
-990.9 -111.2
-989.4 -123.5
-987.8 -135.8
-986.0 -148.0
-984.1 -160.3
-982.1 -172.5
-979.9 -184.6
-977.5 -196.8
-975.0 -208.9
-972.3 -221.0
-969.5 -233.1
-966.5 -245.1
-963.4 -257.1
-960.1 -269.0
-956.7 -280.9
-953.1 -292.8
-949.4 -304.6
-945.6 -316.4
-941.6 -328.1
-937.4 -339.7
-933.1 -351.3
-928.7 -362.9
-924.1 -374.4
-919.4 -385.9
-914.5 -397.2
-909.5 -408.6
-904.4 -419.8
-899.1 -431.0
-893.7 -442.2
-888.1 -453.2
-882.4 -464.2
-876.6 -475.1
-870.6 -486.0
-864.5 -496.8
-858.3 -507.5
-851.9 -518.1
-845.4 -528.6
-838.8 -539.1
-832.1 -549.4
-825.2 -559.7
-818.2 -569.9
-811.0 -580.1
-803.7 -590.1
-796.4 -600.0
-788.8 -609.9
-781.2 -619.6
-773.5 -629.3
-765.6 -638.8
-757.6 -648.3
-749.5 -657.6
-741.3 -666.9
-732.9 -676.0
-724.5 -685.1
-715.9 -694.0
-707.2 -702.9
-698.5 -711.6
-689.6 -720.2
-680.6 -728.7
-671.5 -737.1
-662.3 -745.4
-653.0 -753.6
-643.6 -761.6
-634.0 -769.5
-624.4 -777.4
-614.7 -785.0
-604.9 -792.6
-595.1 -800.1
-585.1 -807.4
-575.0 -814.6
-564.8 -821.7
-554.6 -828.6
-544.3 -835.4
-533.9 -842.1
-523.4 -848.7
-512.8 -855.1
-502.1 -861.4
-491.4 -867.6
-480.6 -873.6
-469.7 -879.5
-458.7 -885.3
-447.7 -890.9
-436.6 -896.4
-425.4 -901.8
-414.2 -907.0
-402.9 -912.1
-391.6 -917.0
-380.1 -921.8
-368.7 -926.4
-357.1 -930.9
-345.5 -935.3
-333.9 -939.5
-322.2 -943.6
-310.5 -947.5
-298.7 -951.3
-286.8 -954.9
-275.0 -958.4
-263.0 -961.8
-251.1 -965.0
-239.1 -968.0
-227.0 -970.9
-215.0 -973.6
-202.9 -976.2
-190.7 -978.7
-178.6 -981.0
-166.4 -983.1
-154.1 -985.1
-141.9 -986.9
-129.6 -988.6
-117.3 -990.2
-105.0 -991.5
-92.7 -992.8
-80.4 -993.9
-68.0 -994.8
-55.7 -995.5
-43.3 -996.2
-30.9 -996.6
-18.6 -996.9
-6.2 -997.1
6.2 -997.1
18.6 -996.9
30.9 -996.6
43.3 -996.2
55.7 -995.5
68.0 -994.8
80.4 -993.9
92.7 -992.8
105.0 -991.5
117.3 -990.2
129.6 -988.6
141.9 -986.9
154.1 -985.1
166.4 -983.1
178.6 -981.0
190.7 -978.7
202.9 -976.2
215.0 -973.6
227.0 -970.9
239.1 -968.0
251.1 -965.0
263.0 -961.8
275.0 -958.4
286.8 -954.9
298.7 -951.3
310.5 -947.5
322.2 -943.6
333.9 -939.5
345.5 -935.3
357.1 -930.9
368.7 -926.4
380.1 -921.8
391.6 -917.0
402.9 -912.1
414.2 -907.0
425.4 -901.8
436.6 -896.4
447.7 -890.9
458.7 -885.3
469.7 -879.5
480.6 -873.6
491.4 -867.6
502.1 -861.4
512.8 -855.1
523.4 -848.7
533.9 -842.1
544.3 -835.4
554.6 -828.6
564.8 -821.7
575.0 -814.6
585.1 -807.4
595.1 -800.1
604.9 -792.6
614.7 -785.0
624.4 -777.4
634.0 -769.5
643.6 -761.6
653.0 -753.6
662.3 -745.4
671.5 -737.1
680.6 -728.7
689.6 -720.2
698.5 -711.6
707.2 -702.9
715.9 -694.0
724.5 -685.1
732.9 -676.0
741.3 -666.9
749.5 -657.6
757.6 -648.3
765.6 -638.8
773.5 -629.3
781.2 -619.6
788.8 -609.9
796.4 -600.0
803.7 -590.1
811.0 -580.1
818.2 -569.9
825.2 -559.7
832.1 -549.4
838.8 -539.1
845.4 -528.6
851.9 -518.1
858.3 -507.5
864.5 -496.8
870.6 -486.0
876.6 -475.1
882.4 -464.2
888.1 -453.2
893.7 -442.2
899.1 -431.0
-502.8 862.7
