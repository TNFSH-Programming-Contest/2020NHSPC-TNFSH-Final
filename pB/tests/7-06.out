12510982.929
